tools/refguide_check.py

```diff
@@ -644,9 +761,23 @@ def _do_check(self, want, got):
 
 
 def _run_doctests(tests, full_name, verbose, doctest_warnings):
-    """Run modified doctests for the set of `tests`.
+    """
+    Run modified doctests for the set of `tests`.
+
+    Parameters
+    ----------
+    tests: list
 
-    Returns: list of [(success_flag, output), ...]
+    full_name : str
+
+    verbose : bool
+
+    doctest_warning : bool
+
+    Returns
+    -------
+    list
+        List of [(success_flag, output), ...]
```

Comment:
I think this is wrong, could you check what it really returns?

Reply:
Rectified
Changed to tuple(bool, list) -> Tuple of (success, output)